lib/meta.mjs

```diff
@@ -1 +1,10 @@
-export { version } from "../package.json" with { type: "json" };
```

Comment:
i think we can simply change it to `import pkg; export const version of pkg.version`